kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/AuxiliaryVoucherValidation.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+ Copyright 2005-2008 The Kuali Foundation
+ 
+ Licensed under the Educational Community License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.opensource.org/licenses/ecl2.php
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+  <bean id="AuxiliaryVoucher-accountingLineValuesAllowedValidation" class="edu.arizona.kfs.sys.document.validation.impl.AccountingLineValuesAllowedValidationHutch" abstract="true">

```

Comment:
I think you're right. In  v3, the behavior is really odd given that in some spots spring changes are directly in the kuali package, while others are properly overridden in the az package. When I load the v3 spring context in Intellij, and follow references for the above, I see a kuali spring file that points to the az bean we see here -- this shouldn't ever happen, and shows how v3 has some undesirable behavior. I'll delete this file, since this is already provided in the system validators as you observed (and as I verified in the spring explorer).